hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/common/statemachine/commandhandler/CreatePipelineCommandHandler.java

```diff
@@ -108,9 +108,13 @@ public void handle(SCMCommand command, OzoneContainer ozoneContainer,
             pipelineID);
       }
     } catch (IOException e) {
-      LOG.error("Can't create pipeline {} {} {}",
-          createCommand.getReplicationType(),
-          createCommand.getFactor(), pipelineID, e);
+      // The server.addGroup may exec after a getGroupManagementApi call
+      // from another peer, so we may got an AlreadyExistsException.
+      if (!(e.getCause() instanceof AlreadyExistsException)) {
+        LOG.error("Can't create pipeline {} {} {}",
+            createCommand.getReplicationType(),
+            createCommand.getFactor(), pipelineID, e);
+      }
```

Comment:
```suggestion
      } else { 
          LOG.info("pipelint {} has already exist", pipelineID);
      }
```
better to log this issue

Reply:
@JacksonYao287 hmmm, this might be slightly different from the case in HDDS-5726, because the duplicate handling is intended behavior, and we "// do not log" in the rpc reply handling code so we do not log here either.